dbt_worker/tasks.py

```diff
@@ -103,9 +128,53 @@ def _insert_log_path(command: List[str], task_id: str):
         return
     command.insert(0, LOG_PATH_ARGS)
     command.insert(1, get_task_artifacts_path(task_id, None))
+    command.insert(2, LOG_FORMAT_ARGS)
+    command.insert(3, LOG_FORMAT_DEFAULT)
+
+
+def _is_command_has_project_dir(command: List[str]) -> bool:
```

Comment:
FYI no changes were made to `_is_command_has_project_dir` `resolve_project_dir` or `append_project_dir` apart from moving them and making the latter 2 public